### PR TITLE
Set width of columns in file table (#3453)

### LINKED ISSUE #3453: Table columns in "Download Files" section are disproportionate
The table columns in the "Download Files" section of a project page are not proportioned well. Filename, Size & Hash are all crammed into a single tiny column, while three less important things (File Type, Python Version, and Upload Date) take up 75% of the width.

I am not able to visually scan for the files I want because the filenames are so wrapped.

Case in point: https://pypi.org/project/numpy/#files
Compare to: https://pypi.python.org/pypi/numpy

One potential solution could be to put data on two lines, something like this:

<table>
  <tr>
    <th>Filename</th>
    <th>Size</th>
    <th>Hash</th>
    <th>Type</th>
    <th>Python Version</th>
    <th>Upload Date</th>
  </tr>
  <tr>
    <td colspan="6">numpy-1.14.2-cp27-cp27m-macosx_10_6_intel.macosx_10_9_intel.macosx_10_9_x86_64.macosx_10_10_intel.macosx_10_10_x86_64.whl</td>
  </tr>
  <tr>
    <td></td>
    <td>4.7 MB</td>
    <td>SHA256</td>
    <td>Wheel</td>
    <td>cp27</td>
    <td>Mar 12, 2018</td>
  </tr>
  <tr>
    <td colspan="6">numpy-1.14.2-cp27-cp27m-manylinux1_i686.whl</td>
  </tr>
  <tr>
    <td></td>
    <td>8.7 MB</td>
    <td>SHA256</td>
    <td>Wheel</td>
    <td>cp27</td>
    <td>Mar 12, 2018</td>
  </tr>
  <tr>
    <td colspan="6">numpy-1.14.2-cp27-cp27m-manylinux1_x86_64.whl</td>
  </tr>
  <tr>
    <td></td>
    <td>12.1 MB</td>
    <td>SHA256</td>
    <td>Wheel</td>
    <td>cp27</td>
    <td>Mar 12, 2018</td>
  </tr>
</table>

### PR DESCRIPTION
**Pull request by yeraydiazdiaz**
Fixes #3453 

screen shot 2018-04-05 at 14 00 59

### CODE REVIEW

**Comment by nlhkabu** [comment]
Hi @yeraydiazdiaz thanks for this PR.  Instead of hard-coding the style, you can actually control this width here:

https://github.com/pypa/warehouse/blob/master/warehouse/static/sass/blocks/_table.scss#L75-L141

If you could also check the results on a range of resolutions, that would be great :)

File: docker-compose.yml [hunk 1]
```diff
@@ -5,7 +5,7 @@ services:
     image: postgres:10.1
     ports:
       # 5432 may already in use by another PostgreSQL on host
-      - "5433:5432"
+      - "5439:5432"
```
Comment (di) [inline docker-compose.yml]: Could you revert this change?
